src/features/logical-assignment.md

```diff
@@ -89,6 +89,6 @@ It may help to think about the symmetry with other compound assignment operators
 
 <feature-support chrome="85"
                  firefox="79 https://bugzilla.mozilla.org/show_bug.cgi?id=1629106"
-                 safari="partial https://webkit.org/blog/10428/release-notes-for-safari-technology-preview-105/"
+                 safari="yes https://developer.apple.com/documentation/safari-release-notes/safari-14-beta-release-notes#New-Features:~:text=Added%20logical%20assignment%20operator%20support."
```

Comment:
What's the rule for when a browser feature support should be a version number vs `"yes"`?

Reply:
Ah, good point, it should be a version number except when the exact version is unknown — I just forgot to list 14 in this case. Will change — thanks!